managers/commons/constants.py

```diff
@@ -1 +1,3 @@
 MAX_NAME_LENGTH = 64
+CORRECT_DATE_FORMAT = "%Y-%m-%d"
+MESSAGE_FOR_CORRECT_DATE_FORMAT = "Please enter date in this format: YYYY-MM-DD"
```

Comment:
The case is that now `MESSAGE_FOR_CORRECT_DATE_FORMAT` will not get updated when `CORRECT_DATE_FORMAT` changes, which was a little bit my point here. But message "Please enter date in this format: %Y-%m-%d" might look weird. So we either do it one way or another. What do you guys think? @Karrp @kbeker

Reply:
@rwrzesien As far as I understand, this PR should add some hint for user how should he put correct date, right? I see that you are talking here about possible out-dating code. Maybe we should add here simply `assert` to constrain remember about making changes with this constants, or system-check, but about this I think it would be over-engineering?

Reply:
In my opinion this format `%Y-%m-%d` looks very weird so I decided to make it in this way. Yes it will not be updated, however we have to add migrations after each update, it could be a small inconvenience, isn't it? I found some solution, here -> https://github.com/tutorcruncher/django-bootstrap3-datetimepicker.
We could write our date format like that `YYYY-MM-DD` and then `BasePickerInput` convert it to this `%Y-%m-%d`
But if we have help_text in models then we still need migrations after each update.

Reply:
@kbeker 
Yes, definitely system-check would be too much, but about `assert` how would you do it without over-complicating it? My point is that the `CORRECT_DATE_FORMAT` might change but `MESSAGE_FOR_CORRECT_DATE_FORMAT` forgot to be updated. The whole pull request is about reusing same date format in several places.

@Szymiks 
I like the solution, if it works correctly with bootstrap datepicker then it is OK for me. Just change to formatted string in one place and I think we are good to go.